### PR TITLE
updates for css3 media queries article, now in proper repo

### PR DESCRIPTION
**Pull request by StraubJones**
Adds some more examples, info, and a few helpful links to the CSS3 media queries article, mostly from my old PR https://github.com/freeCodeCamp/guide/pull/4136

I guess between the time I wrote it a year ago and it got reviewed another media queries article was added, and I missed the first email regarding the merge conflicts. Most of the new article seems better than mine was anyway for beginners, I can admit 😄 but I still think some of the further info and links are helpful, so people realize that media queries can do more than just basic responsive web design.

Also searched through current PRs and no pending work on this article seems to have been done, so it should avoid the merge conflicts this time.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: client/src/pages/guide/english/css/css3-media-queries/index.md [hunk 1]
```diff
@@ -50,8 +50,39 @@ The `p` tag will have a padding of 30px as soon as the screen reaches min 768px
...
+      /* ... */
+    }
+```
+
 ### More Information
 * [MDN - media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
 * [W3 Schools - @media rule](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp)
 * [CSS Tricks Standard Device Widths Atricle](https://css-tricks.com/snippets/css/media-queries-for-standard-devices/)
 * [https://alistapart.com/article/responsive-web-design](Ethan Marcotte A List Apart Atricle on  Responsive Web Design)
+* [http://bradfrost.com/blog/post/7-habits-of-highly-effective-media-queries/](Brad Frost 7 habits of highly effective media queries)
```
Comment (ezioda004) [inline client/src/pages/guide/english/css/css3-media-queries/index.md]: The links are not formatted correctly, the text should be in the square bracket and the link should be in the parentheses. Can you fix it?

File: client/src/pages/guide/english/css/css3-media-queries/index.md [hunk 2]
```diff
@@ -50,8 +50,39 @@ The `p` tag will have a padding of 30px as soon as the screen reaches min 768px
...
+* [https://www.templatemonster.com/blog/css-media-queries-for-all-devices-and-browsers-including-ie7-and-ie8/](How to make media queries work on older browsers)
```
Comment (ezioda004) [inline client/src/pages/guide/english/css/css3-media-queries/index.md]: Here as well.

**Review by ezioda004 — CHANGES_REQUESTED** [review]
Please check the review and make the required changes.

File: client/src/pages/guide/english/css/css3-media-queries/index.md (hunk 1 above)
Comment (StraubJones) [reply client/src/pages/guide/english/css/css3-media-queries/index.md]: Ok, I'll get right on it.

**Review by ezioda004 — APPROVED** [review]
LGTM 👍

**Comment by ezioda004** [comment]
Thank you @StraubJones for your contribution and congratulations on your first PR merge to this repo. 🎉

**Pull request merged**